### PR TITLE
"Peng" goes between "Kopi" and "Gah Dai", not at the end

### LINKED ISSUE #13: Order for 'peng' is wrong
It should be Kopi Peng Gah Dai instead of Kopi Gah Dai Peng. Not sure for other expressions tho'.

@chuyeow @soulofpeace any ideas?

### PR DESCRIPTION
**Pull request by rctay**
Fixes #13.

Also applies to Kopi O and Kopi C.

### CODE REVIEW

File: lib/kopi.coffee [hunk 1]
```diff
@@ -113,12 +113,16 @@ class Kopi
     else
       coffeeLevel = ' Po'
 
-    if i.ice
-      ice = ' Peng'
-    else
-      ice = ''
+    defaultTemplate = "Kopi#{milk}#{coffeeLevel}#{sugarLevel}"
 
-    "Kopi#{milk}#{coffeeLevel}#{sugarLevel}#{ice}"
```
Comment (cheeaun) [inline lib/kopi.coffee]: Would be great if `{ice}` stays in the string :smile:

**Comment by rctay** [comment]
Kept `#{ice}` like you said @cheeaun !

**Comment by cheeaun** [comment]
Closing this, now that the latest version has both `Peng` and `Pua Sio`.

**Comment by rctay** [comment]
:+1:

**Pull request closed without merging**